client/my-sites/customer-home/cards/features/help-search/index.jsx

```diff
@@ -95,11 +95,11 @@ const HelpSearch = ( { searchQuery, openDialog, track } ) => {
 					</div>
 				</div>
 				<div className="help-search__footer">
-					<a className="help-search__cta" href="/help">
+					<a className="help-search__cta" href="/help/contact">
 						<span className="help-search__help-icon">
 							<Gridicon icon="help" size={ 36 } />
 						</span>
-						{ translate( 'More help' ) }
+						{ translate( 'Contact Support' ) }
```

Comment:
Minor, but I would lowercase "support" here; we've moved away from title case for buttons. Good explanation in #39692

Reply:
Updated! :)